web/permissions/permissions.go

```diff
@@ -278,6 +300,20 @@ func patchPermission(getPerms getPermsFunc, paramName string) echo.HandlerFunc {
 			}
 		}
 
+		// Handle metadata
+		// If the metadata has been given in the body request, just apply it to
+		// the patch
+		if patch.Metadata != nil {
```

Comment:
I'd have added those lines to keep the createdAt from the old metadata if the new metadata doesn't say anything about that (and for some other fields):

```go
if toPatch.Metadata != nil {
    patch.Metadata.EnsureCreatedFields(toPatch.Metadata)
}
```